### DISCUSSION TITLE
Jellyseerr integration.

### DISCUSSION THREAD

**Discussion by xawoger** [hrfee/jfa-go#371] [Q&A]
I am confused because it is mentioned in this project ( https://github.com/hrfee/jfa-go/pull/351 ) that this integration is supported but there is no setting in it to integrate jellyseerr. I have to mention that I am running stable/latest version.Any ideas?

**Comment by hrfee** [comment]
You'll have to use the unstable version for now. The feature itself is fine, I just haven't gotten around to releasing a stable build in a while since there's more things i'd like to add.

**Comment by gomaaz** [comment]
is it possible to also delete users with this integration?
I noticed: deleted users in jfa keeps existing in jellyseerr... 

thanks in advance

**Reply by hrfee** [reply]
I didn't implement this as Jellyfin users who log in to Jellyseerr are classed as "Jellyfin User"s (by default), which derive their validity from Jellyfin and so deleting the account through jfa and therefore on Jellyfin should stop them from logging in. In Jellyseerr's User list, is the type of the deleted user "Jellyfin User" or something else? The type should only change if they/you try to change their password through Jellyseerr, which shouldn't be done.
